service/telemetry/internal/migration/normalize.go

```diff
@@ -0,0 +1,10 @@
+package migration
+
+import "strings"
+
+func normalizeEndpoint(endpoint string) *string {
+	if !strings.HasPrefix(endpoint, "https://") && !strings.HasPrefix(endpoint, "http://") {
+		endpoint = "http://" + endpoint
```

Comment:
If someone uses grpc instead of http/proto will this break them?

Reply:
no, the same logic was applied to both grpc + http in the old code path, i tested it locally to ensure it works